dht_bootstrap.go

```diff
@@ -189,9 +189,6 @@ func (dht *IpfsDHT) Bootstrap(_ context.Context) error {
 // error and close. The channel is buffered and safe to ignore.
 func (dht *IpfsDHT) RefreshRoutingTable() <-chan error {
 	res := make(chan error, 1)
-	select {
-	case dht.triggerRtRefresh <- res:
-	default:
-	}
+	dht.triggerRtRefresh <- res
```

Comment:
is the `triggerRtRefresh` channel buffered?

Reply:
It's not. But, the reader drains all the reqs on the channel before proceeding so it can signal the result of the refresh to all "simultaneous" callers.

However, what could happen here is that if a refresh is already in progress, all callers of `RefreshRoutingTable `will block till the current refresh round finishes.

@Stebalien Are we okay with that ?

Reply:
That's the intended behavior, at least for the moment. I want to be able to:

1. Connect to a peer.
2. Trigger a refresh that uses that peer.

That means I want the refresh to start _after_ I call the function and I don't want to count any refreshes that started before.